### PR TITLE
Miscellaneous cleanup

### PR DESCRIPTION
**Pull request by parlough**
Very minimal miscellaneous cleanup.

### CODE REVIEW

**Review by RedBrogdon — APPROVED** [review]
LGTM. I'd love for @johnpryan to have a look as well.

File: lib/services/execution.dart [hunk 1]
```diff
@@ -22,7 +20,7 @@ abstract class ExecutionService {
     String html,
     String css,
     String javaScript, {
-    String modulesBaseUrl,
+    String /*?*/ modulesBaseUrl,
```
Comment (johnpryan) [inline lib/services/execution.dart:23]: I don't see these hints committed very often. Is the idea here to start preparing this package for null safety?

**Review by johnpryan — APPROVED** [review]
These look great, just curious why there's a`/*?*/` hint.

File: lib/services/execution.dart (hunk 1 above)
Comment (parlough) [reply lib/services/execution.dart:23]: It's something I've picked up from @srawlins as I'm modifying old code. They are hints for the migration tool and can be useful for just viewing the code as well before we migrate.

Comment (johnpryan) [reply lib/services/execution.dart:23]: SGTM

**Pull request merged**